src/baldr/CMakeLists.txt

```diff
@@ -69,8 +84,9 @@ set(sources
     verbal_text_formatter_us_tx.cc
     verbal_text_formatter_factory.cc)
 
-#basic timezone stuff
 list(APPEND sources
+    ${CMAKE_CURRENT_BINARY_DIR}/traffic_tile_version.h
```

Comment:
should we use `GENERATED_VERSION_HEADER` here as well?

Reply:
Yup, good catch!